tripper/triplestore.py

```diff
@@ -294,33 +294,35 @@ def parse(
         self,
         source=None,
         format=None,
-        backend=None,
-        backend_kwargs=None,
+        fallback_backend="rdflib",
+        fallback_backend_kwargs=None,
```

Comment:
Since we didn't need to change backend/backend_kwargs to fallback_backend/fallback_backend_kwargs in the tests, it seems that we do not have adeguate tests.

Reply:
Well, the collection backend does not provide a parse() method, so it will use the fallback backend. That is tested in tests/backends/test_collection.py.